File: src/main/java/ourvillagerdiscounts/ourvillagerdiscounts/mixin/VillagerGossipEntriesInvoker.java
package ourvillagerdiscounts.ourvillagerdiscounts.mixin;

import net.minecraft.village.VillagerGossips;
import org.spongepowered.asm.mixin.Mixin;
import org.spongepowered.asm.mixin.gen.Invoker;

import java.util.stream.Stream;

@Mixin(VillagerGossips.class)
public interface VillagerGossipEntriesInvoker {
    @Invoker("entries")
    Stream<VillagerGossips.GossipEntry> invokeEntries();
}

File: src/main/kotlin/ourvillagerdiscounts/ourvillagerdiscounts/mixin/VillagerGossipEntriesInvoker.kt
package ourvillagerdiscounts.ourvillagerdiscounts.mixin

import org.spongepowered.asm.mixin.Mixin
import net.minecraft.village.VillagerGossips
import net.minecraft.village.VillagerGossips.GossipEntry
import org.spongepowered.asm.mixin.gen.Invoker
import java.util.stream.Stream

@Mixin(VillagerGossips::class)
interface VillagerGossipEntriesInvoker {
    @Invoker("entries")
    fun invokeEntries(): Stream<GossipEntry>
}